Automattic/wp-calypso
# Design Picker: Redirect Partner Themes purchases to the Thank You page

Fixes https://github.com/Automattic/wp-calypso/issues/81604

## Proposed Changes

Redirect the user to the Thank You page after purchasing a Partner Theme during onboarding.

https://github.com/Automattic/wp-calypso/assets/5039531/350fa408-c58d-41f1-812b-55103fc7c28e



## Testing Instructions

* Apply the diff D118900-code to your sandbox and direct requests to the sandbox
* Go to the Design Picker page with a Marketplace theme selected `/setup/update-design/designSetup?siteSlug=:site&theme=:theme_slug`. Theme slugs such as `makoney` and `olsen-fse` can be used.
* Click on `Unlock Theme`
* You should see the modal with the updated layout.
* Click to proceed and complete the purchase
* After completing the purchase you should be redirected to the Thank You page to activate the theme.
* Test with atomic and non-atomic sites

<img width="1231" alt="CleanShot 2023-09-14 at 14 02 54@2x" src="https://github.com/Automattic/wp-calypso/assets/5039531/792dfdca-401e-4180-a67f-f13b2c66f2aa">



## Pre-merge Checklist
